source/security/private-endpoints.txt

```diff
@@ -100,111 +93,82 @@ connection using either the App Services UI or the Admin API.
 
          .. step:: Create an Interface Endpoint in Your VPC
```

Comment:
... should I know what an Interface Endpoint is?

does it assume a specific endpoint type requirement? if it _has_ to be interface, why do i have to get it in the private endpoint svcs response (and the example output shows a "general" type)??

Reply:
Yeah, good point. We don't need this to be so specific.

Reply:
Actually, looking it over again, I do think we need to include "interface" here. I've simplified a bit to "Create a VPC Interface Endpoint".

The UI uses "VPC Interface Endpoint" and it's a helpful distinction to avoid confusion between the VPC endpoint and the Private Endpoint users are creating.

But maybe I didn't totally understand your question here.